.cirrus.yml

```diff
@@ -213,6 +213,18 @@ opensuse_tumbleweed_task:
     << : *RESOURCES_TEMPLATE
   << : *CI_TEMPLATE
 
+ubuntu2210_task:
+  container:
+    # Ubuntu 22.10 EOL: July 2023
+    dockerfile: ci/ubuntu-22.10/Dockerfile
+    << : *RESOURCES_TEMPLATE
+  << : *CI_TEMPLATE
+  env:
+    ZEEK_CI_CREATE_ARTIFACT: 1
+  upload_binary_artifacts:
+    path: build.tgz
+  benchmark_script: ./ci/benchmark.sh
```

Comment:
Do we want to duplicate the benchmarking from `ubuntu22_task`? I'm not sure what's guided this in the past — @timwoj might know best.

Reply:
No, it should only be in the ubuntu22 task.

Reply:
ups, sorry about that.